doc/troubleshooting.md

```diff
@@ -68,4 +68,8 @@ If the operator is running fine, check the status of the `RuntimeComponent` Cust
 # Known Issues
 
 Operator might crash on startup when optional CRDs API group (eg. serving.knative.dev/v1alpha1) is 
```

Comment:
Unrelated to this PR. I think we can remove this or state that it's an issue in releases prior to 0.5.0.